scrapers/declension.py

```diff
@@ -0,0 +1,166 @@
+import re
+from urllib.parse import quote
+import requests
+from utils import send_request_with_retries
+from bs4 import BeautifulSoup
+
+
+api_url = "https://sloworz.org/api/graphql"
```

Comment:
```suggestion
API_URL = "https://sloworz.org/api/graphql"
```
https://peps.python.org/pep-0008/#constants